sycl/source/detail/scheduler/scheduler.hpp

```diff
@@ -19,6 +19,152 @@
 #include <set>
 #include <vector>
 
+/// \defgroup sycl_graph DPC++ Execution Graph
+///
+/// DPC++, unlike OpenCL, provides a programming model in which user doesn't
+/// need to manage dependencies between kernels and memory explicitly. The DPC++
+/// Runtime must ensure correct execution with respect to order commands are
+/// submitted.
+///
+/// This document describes the part of the DPC++ Runtime that is responsible
```

Comment:
This document -> this source? :)

Reply:
Hm... This is going to be rendered to Doxygen. I think `document` is more appropriate word here, as it is both clear when reading the source and online documentation.